src/test/lib/optimizer/strategy/column_pruning_rule_test.cpp

```diff
@@ -440,8 +440,9 @@ TEST_F(ColumnPruningRuleTest, AnnotatePrunableJoinInput) {
 
       apply_rule(rule, lqp);
 
-      EXPECT_TRUE(join_node->prunable_input_side().has_value()) << "With JoinMode::" << join_mode;
-      EXPECT_EQ(*join_node->prunable_input_side(), prunable_input_side) << "With JoinMode::" << join_mode;
+      SCOPED_TRACE("With JoinMode::" + std::string{magic_enum::enum_name(join_mode)});
```

Comment:
On purpose? If so, what are you using it for here?

Reply:
On purpose to have the `With JoinMode::` stuff once. I think it's okay soince we look over multiple join modes and if we mess up sth in the rule that might break the behavior for one of them. If so, we directly know in which case (we don't see it from the `EXPECT`).